tests/taskcluster/bin/raptor_graph

```diff
@@ -23,6 +23,9 @@ var originalHeadRev = process.env.GITHUB_HEAD_REV;
 
 const TC_CLIENT_OPTS = { timeout: 30 * 1000 };
 
+// taskcluster worker type for raptor tasks
+RAPTOR_WORKER_TYPE = 'raptor-single-capacity';
```

Comment:
ah, probably because no platform is being specified it uses worker type or something else as the default to group it under.  Probably just best to leave it this way :)